lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -329,8 +394,10 @@ def assemble(self, json_data: Dict[AnyStr, Any]) -> Dict[AnyStr, Any]:
             A dict describing the URI path, the json body, and query parameters
             to pass to Elasticsearch:
                 path: The path part of the Elasticsearch URI
-                json: The JSON query to pass to Elasticsearch
-                params: Query parameters to pass to Elasticsearch
+                kwargs: A kwargs dict for the requests API; e.g.,
```

Comment:
Yes, I took Webb's suggestion to break the assembler return into a two-level dict to avoid the need to remove the path from the kwargs.